doc/conf.py

```diff
@@ -17,11 +17,17 @@
 # -- Project information -----------------------------------------------------
 
 project = 'SmartSim'
-copyright = '2021, Hewlett Packard Enterprise'
-author = 'HPE AI & Advanced Productivity'
+copyright = '2022, Hewlett Packard Enterprise'
```

Comment:
I think we could write `2021-2022, Hewlett Packard Enterprise`, to indicate the year the process went public.

Reply:
done